addon/commands/insert-xml-command.ts

```diff
@@ -22,7 +22,8 @@ export default class InsertXmlCommand extends Command {
     const range = selection.lastRange;
     const parsed = parseXml(xml);
     this.model.change(mutator => {
-      mutator.insertNodes(range, parsed.root);
+      const newRange = mutator.insertNodes(range, parsed.root);
+      mutator.selectRange(newRange);
```

Comment:
for my understanding, it's now expected that a command explicitly selects something or do we still (also) provide a "best guess" selection?

Reply:
yes, thats what insertNodes returns
but the command has to decide to use it indeed, it is not automatically applied